diff --git a/modules/Package.swift b/modules/Package.swift
index 479613c0..20f56eae 100644
--- a/modules/Package.swift
+++ b/modules/Package.swift
@@ -104,7 +104,7 @@ let package = Package(
         .package(url: "https://github.com/pointfreeco/swift-case-paths", from: "1.7.2"),
         .package(url: "https://github.com/pointfreeco/swift-url-routing", from: "0.6.2"),
         .package(url: "https://github.com/zcash-hackworks/MnemonicSwift", from: "2.2.5"),
-        .package(url: "https://github.com/valargroup/zcash-swift-wallet-sdk.git", branch: "shielded-vote"),
+        .package(name: "zcash-swift-wallet-sdk", path: "../../zcash-swift-wallet-sdk"),
         .package(url: "https://github.com/flexa/flexa-ios.git", exact: "1.1.4"),
         .package(url: "https://github.com/pacu/zcash-swift-payment-uri", from: "1.0.1"),
         .package(url: "https://github.com/airbnb/lottie-spm.git", from: "4.5.2"),
