diff --git a/modules/Package.swift b/modules/Package.swift
index f4321916..9677e7b0 100644
--- a/modules/Package.swift
+++ b/modules/Package.swift
@@ -105,12 +105,12 @@ let package = Package(
         .package(url: "https://github.com/pointfreeco/swift-case-paths", from: "1.7.2"),
         .package(url: "https://github.com/pointfreeco/swift-url-routing", from: "0.6.2"),
         .package(url: "https://github.com/zcash-hackworks/MnemonicSwift", from: "2.2.5"),
-        .package(url: "https://github.com/valargroup/zcash-swift-wallet-sdk.git", branch: "denver-demo"),
+        .package(name: "zcash-swift-wallet-sdk", path: "../../zcash-swift-wallet-sdk"),
         .package(path: "../vendor/flexa-ios"),
         .package(url: "https://github.com/pacu/zcash-swift-payment-uri", from: "1.0.1"),
         .package(url: "https://github.com/airbnb/lottie-spm.git", from: "4.5.2"),
         .package(url: "https://github.com/KeystoneHQ/keystone-sdk-ios/", from: "0.0.1"),
-        .package(url: "https://github.com/valargroup/BigDecimal.git", branch: "fix/uncomment-platforms"),
+        .package(path: "../../BigDecimal"),
         .package(url: "https://github.com/siteline/swiftui-introspect", from: "26.0.0"),
         .package(url: "https://github.com/pointfreeco/swift-navigation", from: "2.5.1"),
         .package(url: "https://github.com/liamnichols/xcstrings-tool-plugin", from: "1.2.0")
