diff --git a/voting-circuits/Cargo.toml b/voting-circuits/Cargo.toml
index 3612e31..14e4d24 100644
--- a/voting-circuits/Cargo.toml
+++ b/voting-circuits/Cargo.toml
@@ -8,7 +8,7 @@ repository = "https://github.com/valargroup/voting-circuits"
 readme = "README.md"
 
 [dependencies]
-orchard = { version = "=0.13.1", features = ["circuit", "unstable-voting-circuits"] }
+orchard = { path = "../../orchard", features = ["circuit", "unstable-voting-circuits"] }
 halo2_proofs = { version = "0.3", default-features = false, features = ["batch", "floor-planner-v1-legacy-pdqsort"] }
 halo2_gadgets = { version = "0.4", default-features = false }
 pasta_curves = "0.5"
@@ -22,7 +22,7 @@ lazy_static = "1"
 sinsemilla = "0.1"
 
 [dev-dependencies]
-imt-tree = "0.1"
+imt-tree = { path = "../../vote-nullifier-pir/imt-tree" }
 criterion = "0.8"
 rand = "0.8"
 incrementalmerkletree = "0.8.1"
