diff --git a/Cargo.toml b/Cargo.toml
index 22734876..e365d019 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -18,12 +18,19 @@ resolver = "2"
 #   need our governance visibility changes (pub getters on PCZT types, etc.).
 #   Pinned by rev to shielded-vote-main HEAD (d338e04).
 [patch.crates-io]
-orchard = { git = "https://github.com/valargroup/voting-circuits.git", branch = "greg/orchard-0.12" }
+orchard          = { path = "../voting-circuits/orchard" }
+imt-tree         = { path = "../vote-nullifier-pir/imt-tree" }
 
-pczt                = { git = "https://github.com/valargroup/librustzcash.git", rev = "1bc3a279e3c5" }
-zcash_keys          = { git = "https://github.com/valargroup/librustzcash.git", rev = "1bc3a279e3c5" }
-zcash_primitives    = { git = "https://github.com/valargroup/librustzcash.git", rev = "1bc3a279e3c5" }
-zcash_protocol      = { git = "https://github.com/valargroup/librustzcash.git", rev = "1bc3a279e3c5" }
+pczt             = { path = "../librustzcash/pczt" }
+zcash_keys       = { path = "../librustzcash/zcash_keys" }
+zcash_primitives = { path = "../librustzcash/zcash_primitives" }
+zcash_protocol   = { path = "../librustzcash/components/zcash_protocol" }
+
+[patch."https://github.com/valargroup/voting-circuits.git"]
+voting-circuits  = { path = "../voting-circuits/voting-circuits" }
+
+[patch."https://github.com/valargroup/vote-nullifier-pir.git"]
+pir-client       = { path = "../vote-nullifier-pir/pir/client" }
 
 [profile.release]
 lto = true
